src/main/java/htsjdk/samtools/util/Log.java

```diff
@@ -41,13 +41,13 @@
  */
 public final class Log {
     /** Enumeration for setting log levels. */
-    public static enum LogLevel { ERROR, WARNING, INFO, DEBUG }
+    public enum LogLevel { ERROR, WARNING, INFO, DEBUG }
```

Comment:
Inner enums are implicitly static: http://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.9. IntelliJ flagged it.